app/src/main/res/layout/activity_mock_tutor.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove `mock` from the name of this layout. Maybe rename this to `tutor_item_layout`.

Reply:
Done